editor/editor_plugin.cpp

```diff
@@ -654,11 +654,7 @@ void EditorPlugin::make_visible(bool p_visible) {
 }
 
 void EditorPlugin::edit(Object *p_object) {
-	if (p_object->is_class("Resource")) {
-		GDVIRTUAL_CALL(_edit, Ref<Resource>(Object::cast_to<Resource>(p_object)));
-	} else {
-		GDVIRTUAL_CALL(_edit, p_object);
-	}
+	GDVIRTUAL_CALL(_edit, p_object);
```

Comment:
This changes the behavior, no? How does this affect existing plugins and the actual reference counting on that resource?

Reply:
I dont think it is supposed to change behavior, it's passed just the same as every other virtual method taking `Object*` already. If reference count has to increase, it will happen in the callee.